openapi/custom_objects_spec.json

```diff
@@ -352,13 +366,27 @@
             "$ref": "#/definitions/v1.DeleteOptions"
           }
         },
+        {
+          "name": "continue",
+          "uniqueItems": true,
+          "type": "string",
+          "description": "The continue option should be set when retrieving more results from the server. Since this value is server defined, clients may only use the continue value from a previous query result with identical query parameters (except for the value of continue) and the server may reject a continue value it does not recognize. If the specified continue value is no longer valid whether due to expiration (generally five to fifteen minutes) or a configuration change on the server, the server will respond with a 410 ResourceExpired error together with a continue token. If the client needs a consistent list, it must restart their list without the continue field. Otherwise, the client may send another list request with the token received with the 410 error, the server will respond with a list starting from the next key, but from the latest snapshot, which is inconsistent from the previous list results - objects that are created, modified, or deleted after the first list request will be included in the response, as long as their keys are after the \"next key\".\n\nThis field is not supported when watch is true. Clients may start a watch from the last resourceVersion value returned by the server and not miss any modifications.",
+          "in": "query"
+        },
```

Comment:
This is a delete operation, which doesn't support paging. Please move this parameter to `listNamespacedCustomObject`. Same for the `limit` parameter in L383-L389

Reply:
Fixed.